newIDE/app/src/Utils/GDevelopServices/Game.js

```diff
@@ -40,20 +51,15 @@ export type Game = {
   acceptsBuildComments?: boolean,
   acceptsGameComments?: boolean,
   displayAdsOnGamePage?: boolean,
-};
+  cachedCurrentSlug?: CachedGameSlug,
+|};
 
-export type GameCategory = {
+export type GameCategory = {|
```

Comment:
I think the `{||}` are actually giving LESS safety here, because there is no guarantee that the backed will ever return only these fields (this would go against the ability to change these types in the future with addition-only changes).

The advantage of *not* using {| |} is that this will avoid someone in the code to do something like iterating on the keys and assuming they will only receive a specified set of key (that's a stretch, but you get the idea: we don't actually know if the backend can send new properties)

Reply:
interesting. I'll keep the strict typing for the slugs, because I believe the types will not change, or if they do we will update them. I'll remove it for others